recipe/build.sh

```diff
@@ -7,7 +7,6 @@ then
     # make sure cmake can compile openmp code with clang
     export DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib
     clang_version=`${CC} --version | grep "clang version" | cut -d " " -f 3`
-    ln -s $PREFIX/lib/clang/*/include/omp.h $BUILD_PREFIX/lib/clang/${clang_version}/include/omp.h
 fi
```

Comment:
Do we still need this `if`?

Reply:
Maybe not? This recipe is old so it might be good to try without. Also, the R build doesn't seem to be linking to the openmp libraries.

Reply:
FWIW I'm more interested in seeing if we can get this to pass again. Just thinking this might be worth dropping assuming we can accomplish that.

Reply:
Handling in PR ( https://github.com/conda-forge/xgboost-feedstock/pull/38 ).